Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -5,36 +5,54 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public struct DefaultSDKRuntimeConfiguration: SDKRuntimeConfiguration {
+public struct DefaultSDKRuntimeConfiguration<DefaultSDKRuntimeRetryStrategy: RetryStrategy,
```

Comment:
- Added retry-related params to this type
- Extracted default values into static methods